zerver/lib/upload.py

```diff
@@ -194,13 +194,33 @@ def get_total_uploads_size_for_user(user: UserProfile) -> int:
         total_quota = 0
     return total_quota
 
+def get_total_uploads_size_for_realm(realm: Realm) -> int:
+    uploads = Attachment.objects.filter(realm=realm)
+    total_quota = uploads.aggregate(Sum('size'))['size__sum']
```

Comment:
I would write this as 
`used_space = Attachment.objects.filter(realm=realm).aggregate(...)['size_sum']`
i.e. we generally don't add variables that are only used once. 

The name of this function can be `currently_used_upload_space`